lyluatex.sty

```diff
@@ -21,6 +21,7 @@
     ['cleantmp'] = 'false',
     ['current-font-as-main'] = 'true',
     ['fullpage'] = 'false',
+    ['fullpagestyle'] = 'plain',
```

Comment:
I am thinking about how we could leave this issue as primitive as possible (but still satisfactory) in `lyluatex` and do the more complex stuff in `musicexamples`.

I will have to rewrite that pretty much anyway, but the full page example commands in `musicexamples` have a few features that I would on the one hand want to see in our system but on the other hand not duplicated:

* Start the score on the next, the next odd or the next even page (e.g. in a book I may want a two-page score to be printed on opposite pages without page turn).
* Don't interrupt the typesetting, that is, the text is printed continuously until the page break where the score will appear (`\afterpage`).
* Print the caption in the header/footer of the first page (i.e. have different page styles for first and other pages)

If we're doing too much fancy things for the fullpage score it might become more inconvenient to modify the code for the condition that it is used as input for the `musicexamples` command.

Reply:
Actually, for the short-term goal of merging this branch I suggest another modification:

* Use `fullpagestyle` as you have
* Make `default` the default value
* If it is `default` use the page style that is currently in use for regular text (I don't know how that is retrieved but it should be possible, isn't it?)
* Add an option `print-page-number` that allows LilyPond to print page numbers (useful only in case ´fullpagestyle` is set to `empty` (added as 3cba261b)